python/datasources/uhc.py

```diff
@@ -204,18 +202,22 @@ def parse_raw_data(df, breakdown):
                 per_100k_col_name = std_col.generate_column_name(
                     prefix, std_col.PER_100K_SUFFIX)
 
+                # replace extra space to match column correctly
+                df.replace('Voter Participation (Presidential) - Ages 65+ ',
+                           'Voter Participation (Presidential) - Ages 65+', inplace=True)
+
                 if breakdown_value in {'All', 'Total'}:
                     # find row that matches current nested iterations
+
                     matched_row = df.loc[
                         (df[std_col.STATE_NAME_COL] == state) &
                         (df['Measure Name'] ==
                          ALT_ROWS_ALL.get(determinant, determinant))
                     ]
 
-                    # TOTAL voter_participation is avg of pres and midterm data
+                    # TOTAL voter_participation is taken from pres value
                     if determinant in AVERAGED_DETERMINANTS:
```

Comment:
i think we can remove this `AVERAGED_DETERMINANTS` list now, since voting was the only one being averaged and we aren't averaging it anymore. Just treat it as a normal condition, making sure you deal with % vs 100k appropriately